src/core/tracking.js

```diff
@@ -105,8 +105,23 @@ export function getAction(type) {
   }[type] || TRACKING_TYPE_INVALID;
 }
 
+export function doNotTrackEnabled({
+  _navigator = typeof navigator !== 'undefined' ? navigator : null,
+  _window = typeof window !== 'undefined' ? window : null,
+} = {}) {
+  if (!_navigator || !_window) {
+    return false;
+  }
+
+  const dnt = _navigator.doNotTrack || _window.doNotTrack;
```

Comment:
IE 9/10 uses `navigator.msDoNotTrack` - it might be nice to add that for completeness.